web/src/pages/Profile/index.tsx

```diff
@@ -0,0 +1,248 @@
+import Link from "next/link";
+import { SetStateAction, useState } from "react";
+import { MdAccountCircle, MdAddCircle } from "react-icons/md";
+
+import Button from "../../components/Inputs/Button";
+import LinkIconButton from "../../components/Inputs/LinkIconButton";
+import SearchBar from "../../components/Inputs/SearchBar";
+import ImageCard from "../../components/Surfaces/ImageCard";
+
+/*
+ *
+ * Note to viewer:
+ * Please bully, I must learn the error in my ways.
+ *
+ * */
```

Comment:
Don't use comments to talk to the reviewer; instead, include context in the PR comments.